examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,435 @@
+                                    .finally(() => setWaitingForUser(false));
+                            }
+                        }}
+                    >
+                        Register File Hash
+                    </button>
+                )}
+            </label>
+            {activeConnection && activeConnectedAccount && (
+                <p>
+                    {isRegisterFilePage && (
+                        <>
+                            <div>Transaction status{hash === '' ? '' : ' (May take a moment to finalize)'}</div>
+                            {!hash && transactionError && (
+                                <div style={{ color: 'red' }}>Error: {transactionError}.</div>
+                            )}
+                            {!hash && !transactionError && <div className="loadingText">None</div>}
+                            {hash && (
+                                <>
+                                    <button
+                                        className="link"
+                                        type="button"
+                                        onClick={() => {
+                                            window.open(
+                                                `https://testnet.ccdscan.io/?dcount=1&dentity=transaction&dhash=${hash}`,
+                                                '_blank',
+                                                'noopener,noreferrer'
+                                            );
+                                        }}
+                                    >
+                                        {hash}
+                                    </button>
+                                    <br />
+                                </>
+                            )}
+                        </>
+                    )}
+                    {getFileError && <div style={{ color: 'red' }}>Error: {getFileError}.</div>}
+                    {!isRegisterFilePage && witness !== '' && (
+                        <>
+                            <div>On-chain Record:</div>
+                            <div className="loadingText">{witness === null ? 'Not registered' : witness} (witness)</div>
+                            <div className="loadingText">
+                                {timestamp === null ? 'Not registered' : timestamp} (timestamp)
+                            </div>
+                        </>
+                    )}
+                </p>
+            )}
+            <div>
+                <br />
+                Version: {version} |{' '}
+                <a
+                    style={{ color: 'white' }}
+                    href="https://developer.concordium.software/en/mainnet/smart-contracts/tutorials/wCCD/index.html"
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    Learn how to make a wrapper like this
```

Comment:
Is this a wrapper?

Reply:
I will update this text and this link to the written eSealing documentation now:

https://github.com/Concordium/concordium.github.io/pull/727